FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_mutual_auth/DemoTasks/MutualAuthMQTTExample.c

```diff
@@ -456,13 +479,15 @@ static void prvCreateMQTTConnectionWithBroker( MQTTContext_t * pxMQTTContext,
     xTransport.send = TLS_FreeRTOS_send;
     xTransport.recv = TLS_FreeRTOS_recv;
 
-    /* Application callbacks for receiving incoming published and incoming acks
+    /* Application callback for receiving incoming published and incoming acks
      * from MQTT library. */
-    xCallbacks.appCallback = prvEventCallback;
-    xCallbacks.getTime = prvGetTimeMs;
+    xUserCallback = prvEventCallback;
```

Comment:
I was suggested by @aggarg to avoid these stack variables but instead just pass the `prEventCallback` and `prvGetTimeMs` arguments directly in the `MQTT_Init` call...

Reply:
Oh ok. Sure. Will keep it consistent.